lazyflow/operators/tiktorch/classifier.py

```diff
@@ -73,6 +73,33 @@ def input_axes(self):
     def output_axes(self):
         return self.__session.outputAxes
 
+    @property
+    def offset(self):
+        return self.__session.offset
+
+    @property
+    def scale(self):
+        return self.__session.scale
+
+    @property
+    def output_shape(self):
+        """
+        shape = shape(input_tensor) * scale + 2 * offset
+        """
+        offset_by_name = {d.name: d.size for d in self.offset}
+        scale_by_name = {d.name: d.size for d in self.scale}
+
+        result = []
+        for shape in self.__session.validShapes:
+            dim_size_by_name = {d.name: d.size for d in shape.dims}
+            valid_shape = {}
+            for dim in dim_size_by_name:
+                size = int(dim_size_by_name.get(dim, 1) * scale_by_name.get(dim, 1.0) + 2 * offset_by_name.get(dim, 0))
+                valid_shape[dim] = size
+
+            result.append(valid_shape)
+        return result[0]
```

Comment:
This has a nested comprehension, but I think it shows the resulting data structure a bit better.
```suggestion
        offsets = defaultdict({d.name: d.size for d in self.offset}, 0)
        scales = defaultdict({d.name: d.size for d in self.scale}, 1.0)

        output_shapes = [
            {
                dim: int(dim_size * scales[dim] + 2 * offsets[dim])
                for dim, dim_size in shape.dims.items()
            }
            for shape in self.__session.validShapes
        ]
        return output_shapes[0]
```

Reply:
Hm, I have to admit that for me the current version is easier to parse. But I'm gonna add the `defaultdicts` for `scales` and `offsets`!